client/blocks/express-checkout/index.js

```diff
@@ -57,6 +60,7 @@ const expressCheckoutElementsApplePay = ( api ) => ( {
 		} );
 	},
 	paymentMethodId: PAYMENT_METHOD_EXPRESS_CHECKOUT_ELEMENT,
+	gatewayId: 'stripe',
```

Comment:
The changes in this file are not relevant to the modal text, also it does not visibly do anything. I just piggybacked these little changes here. These attributes were missing from the object.